crates/meroctl/src/cli/node.rs

```diff
@@ -0,0 +1,89 @@
+use camino::Utf8PathBuf;
+use clap::{Parser, Subcommand};
+use comfy_table::Table;
+use eyre::eyre;
+use url::Url;
+
+use crate::node_config::{NodeConfig, NodeConnection};
+
+#[derive(Debug, Parser)]
+pub struct AddNodeCommand {
+    /// Name of the node
+    pub name: String,
+
+    /// Path to local node
+    #[arg(long, conflicts_with = "url")]
+    pub path: Option<Utf8PathBuf>,
+
+    /// URL of remote node
+    #[arg(long, conflicts_with = "path")]
+    pub url: Option<Url>,
+
+    /// Authentication key for the node (can also be set via MEROCTL_NODE_KEY env var)
+    #[arg(long, env = "MEROCTL_NODE_KEY")]
```

Comment:
clap already says it

```suggestion
    /// Authentication key for the node
    #[arg(long, env = "MEROCTL_NODE_KEY")]
```